ping_pong test: timeout

https://travis-ci.org/akka/akka-grpc/jobs/365574552
```
[info] Running test ping_pong
[info] [INFO] [04/12/2018 12:06:58.577] [pool-1-thread-1-ScalaTest-running-GrpcInteropSpec] [akka.actor.ActorSystemImpl(default)] Exception thrown, unbinding
[error] org.scalatest.exceptions.TestFailedException
[error] 	at org.scalatest.Assertions.newAssertionFailedException(Assertions.scala:528)
[error] 	at org.scalatest.Assertions.newAssertionFailedException$(Assertions.scala:527)
[error] 	at org.scalatest.WordSpec.newAssertionFailedException(WordSpec.scala:1881)
[error] 	at org.scalatest.Assertions.fail(Assertions.scala:1121)
[error] 	at org.scalatest.Assertions.fail$(Assertions.scala:1117)
[error] 	at org.scalatest.WordSpec.fail(WordSpec.scala:1881)
[error] 	at com.lightbend.grpc.interop.GrpcInteropTests.withGrpcServer(GrpcInteropTests.scala:74)
[error] 	at com.lightbend.grpc.interop.GrpcInteropTests.$anonfun$grpcTests$4(GrpcInteropTests.scala:50)
[error] 	at com.lightbend.grpc.interop.GrpcInteropTests.pendingTestCaseSupport(GrpcInteropTests.scala:86)
[error] 	at com.lightbend.grpc.interop.GrpcInteropTests.$anonfun$grpcTests$3(GrpcInteropTests.scala:49)
[error] 	at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
[error] 	at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
[error] 	at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[error] 	at org.scalatest.Transformer.apply(Transformer.scala:22)
[error] 	at org.scalatest.Transformer.apply(Transformer.scala:20)
[error] 	at org.scalatest.WordSpecLike$$anon$1.apply(WordSpecLike.scala:1078)
[error] 	at org.scalatest.TestSuite.withFixture(TestSuite.scala:196)
[error] 	at org.scalatest.TestSuite.withFixture$(TestSuite.scala:195)
[error] 	at org.scalatest.WordSpec.withFixture(WordSpec.scala:1881)
[error] 	at org.scalatest.WordSpecLike.invokeWithFixture$1(WordSpecLike.scala:1076)
[error] 	at org.scalatest.WordSpecLike.$anonfun$runTest$1(WordSpecLike.scala:1088)
[error] 	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:289)
[error] 	at org.scalatest.WordSpecLike.runTest(WordSpecLike.scala:1088)
[error] 	at org.scalatest.WordSpecLike.runTest$(WordSpecLike.scala:1070)
[error] 	at org.scalatest.WordSpec.runTest(WordSpec.scala:1881)
[error] 	at org.scalatest.WordSpecLike.$anonfun$runTests$1(WordSpecLike.scala:1147)
[error] 	at org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:396)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:384)
[error] 	at org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:373)
[error] 	at org.scalatest.SuperEngine.$anonfun$runTestsInBranch$1(Engine.scala:410)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:384)
[error] 	at org.scalatest.SuperEngine.runTestsInBranch(Engine.scala:379)
[error] 	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:461)
[error] 	at org.scalatest.WordSpecLike.runTests(WordSpecLike.scala:1147)
[error] 	at org.scalatest.WordSpecLike.runTests$(WordSpecLike.scala:1146)
[error] 	at org.scalatest.WordSpec.runTests(WordSpec.scala:1881)
[error] 	at org.scalatest.Suite.run(Suite.scala:1147)
[error] 	at org.scalatest.Suite.run$(Suite.scala:1129)
[error] 	at org.scalatest.WordSpec.org$scalatest$WordSpecLike$$super$run(WordSpec.scala:1881)
[error] 	at org.scalatest.WordSpecLike.$anonfun$run$1(WordSpecLike.scala:1192)
[error] 	at org.scalatest.SuperEngine.runImpl(Engine.scala:521)
[error] 	at org.scalatest.WordSpecLike.run(WordSpecLike.scala:1192)
[error] 	at org.scalatest.WordSpecLike.run$(WordSpecLike.scala:1190)
[error] 	at org.scalatest.WordSpec.run(WordSpec.scala:1881)
[error] 	at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:314)
[error] 	at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:507)
[error] 	at sbt.ForkMain$Run$2.call(ForkMain.java:300)
[error] 	at sbt.ForkMain$Run$2.call(ForkMain.java:290)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] Caused by: java.lang.RuntimeException: java.util.concurrent.TimeoutException: Futures timed out after [3 seconds]
[error] 	at io.grpc.testing.integration2.TestServiceClient.run(TestServiceClient.java:97)
[error] 	at com.lightbend.grpc.interop.AkkaGrpcClientScala.run(AkkaGrpcClientScala.scala:27)
[error] 	at com.lightbend.grpc.interop.GrpcInteropTests.runGrpcClient(GrpcInteropTests.scala:80)
[error] 	at com.lightbend.grpc.interop.GrpcInteropTests.$anonfun$grpcTests$5(GrpcInteropTests.scala:50)
[error] 	at com.lightbend.grpc.interop.GrpcInteropTests.withGrpcServer(GrpcInteropTests.scala:62)
[error] 	... 47 more
[error] Caused by: java.util.concurrent.TimeoutException: Futures timed out after [3 seconds]
[error] 	at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:255)
[error] 	at scala.concurrent.impl.Promise$DefaultPromise.result(Promise.scala:259)
[error] 	at scala.concurrent.Await$.$anonfun$result$1(package.scala:215)
[error] 	at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:53)
[error] 	at scala.concurrent.Await$.result(package.scala:142)
[error] 	at com.lightbend.grpc.interop.AkkaGrpcClientTester.pingPong(AkkaGrpcClientTester.scala:171)
[error] 	at io.grpc.testing.integration2.TestServiceClient.runTest(TestServiceClient.java:142)
[error] 	at io.grpc.testing.integration2.TestServiceClient.run(TestServiceClient.java:93)
[error] 	... 51 more
```

--- a/sbt-plugin/src/sbt-test/gen-scala-server/00-interop/src/test/scala/com/lightbend/grpc/interop/AkkaGrpcClientTester.scala
+++ b/sbt-plugin/src/sbt-test/gen-scala-server/00-interop/src/test/scala/com/lightbend/grpc/interop/AkkaGrpcClientTester.scala
@@ -26,7 +26,7 @@ class AkkaGrpcClientTester(val settings: Settings)(implicit mat: Materializer, e
   private var client: TestServiceClient = null
   private var clientUnimplementedService: UnimplementedServiceClient = null
 
-  private val awaitTimeout = 3.seconds
+  private val awaitTimeout = 7.seconds
 
   def createChannel(): ManagedChannel = ChannelBuilder.buildChannel(settings)
 

